### DISCUSSION TITLE
Why not use SolidStart

### DISCUSSION THREAD

**Discussion by ValBlaze** [AR10Dev/tauri-solid-ts-tailwind-vite#78] [Q&A] [answered]
Is there a reason for not using SolidStart? Is it better to use normal SolidJS to create Tauri apps?

**Comment by AR10Dev** [comment, marked as answer]
Currently, SolidStart is in Beta state and there are a few bugs that need to be solved.
For now, it's better to use SolidJS to create Tauri apps but in the future, SolidStart would be a great option.
If you want to try it, I made a [PR](https://github.com/tauri-apps/create-tauri-app/pull/274) a few months ago, in the [create-tauri-app repository](https://github.com/tauri-apps/create-tauri-app).
In the coming days, I'll publish a template specifically for Tauri and SolidStart
